packages/core/upload/server/bootstrap.js

```diff
@@ -1,21 +1,43 @@
 'use strict';
 
 const { getService } = require('./utils');
+const { ALLOWED_SORT_STRINGS } = require('./constants');
 
 module.exports = async ({ strapi }) => {
-  // set plugin store
-  const configurator = strapi.store({ type: 'plugin', name: 'upload', key: 'settings' });
-
-  // if provider config does not exist set one by default
-  const config = await configurator.get();
-
-  if (!config) {
-    await configurator.set({
-      value: {
+  const Stores = new Map([
+    [
+      'settings',
+      {
         sizeOptimization: true,
         responsiveDimensions: true,
         autoOrientation: false,
       },
+    ],
+    [
+      'view_configuration',
+      {
+        pageSize: 10,
+        sort: ALLOWED_SORT_STRINGS[0],
+      },
+    ],
+  ]);
```

Comment:
What do you think of using an object `defaultConfig` instead of a Map? For code consistency as we use object rather than Map when it's possible